vortex-btrblocks/src/lib.rs

```diff
@@ -127,7 +129,16 @@ pub fn estimate_compression_ratio_with_sampling<T: Scheme + ?Sized>(
     let sample = if is_sample {
         stats.clone()
     } else {
-        stats.sample(64, 10)
+        // We want to sample about 1% of data
+        let source_len = stats.source().len();
+
+        // We want to sample about 1% of data, while keeping a minimal sample of 640 values.
+        let sample_count = usize::max(
+            (source_len / 100) / usize::try_from(SAMPLE_SIZE).vortex_unwrap(),
```

Comment:
could just `(SAMPLE_SIZE as usize)` given that it's const / guaranteed to succeed